bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -213,7 +213,9 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <zetamaxstable               use_biomass_heat_storage=".true." >2.0d00</zetamaxstable>
 <zetamaxstable phys="clm4_5"                                   >2.0d00</zetamaxstable>
 <zetamaxstable phys="clm5_0"                                   >0.5d00</zetamaxstable>
-<zetamaxstable phys="clm5_1"                                   >0.5d00</zetamaxstable>
+<zetamaxstable phys="clm5_1"                                   >2.0d00</zetamaxstable>
+<zetamaxstable phys="clm5_1" z0param_method="ZengWang2007" >0.5d00</zetamaxstable>
+<zetamaxstable phys="clm5_1" z0param_method="Meier2022" >2.0d00</zetamaxstable>
```

Comment:
@ekluzek based on our conversation last week that we want this PR to NOT be answer-changing for tests not using Meier2022, I am reverting line 216 to 0.5, because it fixes the lnd_in files of the Fates tests that differed from baseline (dev129).

Reply:
Yep, we will make Meier2022 the default at a later answer changing tag.